site/content/how-to/traffic-management/securing-backend-traffic.md

```diff
@@ -12,9 +12,13 @@ Learn how to encrypt HTTP traffic between NGINX Gateway Fabric and your backend
 
 In this guide, we will show how to specify the TLS configuration of the connection from the Gateway to a backend pod/s via the Service API object using a [BackendTLSPolicy](https://gateway-api.sigs.k8s.io/api-types/backendtlspolicy/). This covers the use-case where the service or backend owner is doing their own TLS and NGINX Gateway Fabric needs to know how to connect to this backend pod that has its own certificate over HTTPS.
 
+## Note on Gateway API Experimental Features
```

Comment:
It's a little tricky with the current format where its never stated that BackendTLSPolicy is an experimental feature. So this note is here but its never stated. Any thoughts of a better format?

Reply:
Maybe add a callout above this section that states this is an experimental feature?

Reply:
I tried putting the callout above and it looked wonky, the place I put it feels more appropriate. let me know of your thoughts